src/screens/ProfileScreen.js

```diff
@@ -112,18 +110,14 @@ export default function ProfileScreen({ navigation }) {
         />
       ) : (
         <ActivityList
-          addCashAvailable={addCashAvailable}
           header={
             <ProfileMasthead
               addCashAvailable={addCashAvailable}
               onChangeWallet={onChangeWallet}
             />
           }
           isEmpty={isEmpty}
-          isLoading={isLoading}
-          navigation={navigation}
           network={network}
-          recyclerListView={ios}
```

Comment:
We found out on some meeting that there's a code branch in ActivityList that currently never executes, which means that we can skip passing those props.

Reply:
can you clarify?